web-client/integration-tests/petitionsClerkSetsRemoteTrialSessionCalendar.test.js

```diff
@@ -69,23 +79,12 @@ describe('petitions clerk sets a remote trial session calendar', () => {
       loginAs(cerebralTest, 'petitionsclerk@example.com');
       cerebralTest.casesReadyForTrial = [];
       petitionsClerkCreatesNewCase(cerebralTest, fakeFile, trialLocation);
-      petitionsClerkManuallyAddsCaseToTrial(cerebralTest);
+      manuallyAddCaseToTrial(cerebralTest);
     });
   });
 
-  describe('petitions clerk sets calendar for trial session', () => {
-    petitionsClerkViewsNewTrialSession(cerebralTest);
```

Comment:
Why were these test steps removed?

Reply:
I am concerned we lost test coverage in removing these steps, or are the lines that were removed functionality that no longer exists in the system as of implementing this story?

Reply:
I'll make a helper called `petitionsClerkViewsOpenTrialSession` which should make this make more sense